Week2/homework/js-exercises/recipe-ex3.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+let recipe = {
+    mealName : "Mirzaghasemi",
+    serves: 2,
+    ingredients : ["2 eggplants", "2 eggs", "1 garlic"]
+};
+//Method1:
+// for (let property in recipe) {
+//     console.log(property + ": " + recipe[property]);
+// };
+
+//Method2:
+// for (let property in recipe) {
+//     console.log(`${property}: ${recipe[property]}`);
+// };
+
+//Method3:
+let property = Object.keys(recipe);
+let value = Object.values(recipe);
+
+for (let i = 0; i < Object.keys(recipe).length; i++) {
+    
```

Comment:
Object.values is not necessarily sorted. Therefore it is possible that you get [2, 'pizza', [..]] instead. Which means you can possibly get Meal name: 2. You should be careful how you use it.

Reply:
I think I need more explanation! Let's do it in today's meeting